spec/graphql/static_validation/rules/argument_literals_are_compatible_spec.rb

```diff
@@ -18,7 +18,7 @@
   let(:errors) { validator.validate(document) }
 
   it 'finds undefined arguments to fields and directives' do
-    assert_equal(3, errors.length)
+    assert_equal(4, errors.length)
```

Comment:
:+1: I see that I muddled input & result coercion from http://facebook.github.io/graphql/#sec-Boolean